GitUI/NBugReports/BugReportInvoker.cs

```diff
@@ -42,14 +45,17 @@ internal static string Append(StringBuilder text, Exception exception)
 
             if (exception is ExternalOperationException externalOperationException)
             {
+                // Exit code: <n>
+                AppendIfNotEmpty($"{externalOperationException.ExitCode}{Environment.NewLine}", TranslatedStrings.ExitCode);
+
                 // Command: <command>
                 AppendIfNotEmpty(externalOperationException.Command, TranslatedStrings.Command);
 
                 // Arguments: <args>
                 AppendIfNotEmpty(externalOperationException.Arguments, TranslatedStrings.Arguments);
 
                 // Directory: <dir>
-                AppendIfNotEmpty(externalOperationException.WorkingDirectory, TranslatedStrings.WorkingDirectory);
+                AppendIfNotEmpty(externalOperationException.Directory, TranslatedStrings.Directory);
```

Comment:
Why are we departing from the notion of working directory? Git Extensions and git work with working directories, and don't work with arbitrary directories.

Reply:
Reverted this leftover of the planned generalized use of `ExternalOperationException` e.g. for file operations, which might be handled sufficiently since #9094.